checker/tests/fenumswing/IdentityArrayList.java

```diff
@@ -1,10 +1,15 @@
 import java.util.Arrays;
+import org.checkerframework.checker.fenum.qual.FenumTop;
 
 /*
  * This test case violates an assertion in the compiler.
  * It does not depend on the Fenum Checker, it breaks for any checker.
  */
 public class IdentityArrayList {
+    // The type of the third argument to Arrays.copyOf should be:
```

Comment:
Shouldn't this be put into the `https://github.com/typetools/checker-framework/blob/master/checker/src/org/checkerframework/checker/fenum/jdk.astub` file?